fairseq/models/distributed_fairseq_model.py

```diff
@@ -111,7 +114,32 @@ def __getattr__(self, name):
                 return getattr(wrapped_module, name)
             return super().__getattr__(name)
 
-    return _DistributedFairseqModel(**init_kwargs)
```

Comment:
This is almost equivalent to my previous commit:
https://github.com/SciPioneer/fairseq/commit/b4039f563f9935625a515f395b477ec056a31ddd